src/app/components/layouts/overlay/terra-overlay.component.html

```diff
@@ -26,13 +26,16 @@ <h4 class="modal-title">{{inputOverlayTitle}}</h4>
                 
                 <terra-button *ngIf="inputPrimaryButtonInterface"
                               [inputIsAlignRight]="false"
-                              [inputIsSecondary]="true"
                               [inputCaption]="inputPrimaryButtonInterface.caption"
                               [inputTooltipText]="inputPrimaryButtonInterface.tooltipText"
                               inputTooltipPlacement="bottom"
                               [inputIsDisabled]="inputPrimaryButtonInterface.isDisabled"
+                              [inputIsPrimary] = "inputPrimaryButtonInterface.isPrimary"
+                              [inputIsSecondary] = "!inputPrimaryButtonInterface.isPrimary 
```

Comment:
ist zwar Geschmackssache aber kannst du das in einer Variable in die .ts auslagern und die Abfrage dort umsetzen?